src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -39,6 +39,7 @@
 
 public class FlutterSettingsConfigurable implements SearchableConfigurable {
   private static final boolean DISABLE_INSPECTOR = true;
+  public static final boolean WIDGET_FILTERING_ENABLED = false;
```

Comment:
Looks good for getting a sense of how the filtering works. Longer term, do we think this is something the user would want to toggle? If so, it should potentially exist as an option in the overflow menu in the Flutter view.

Reply:
Absolutely.  We're not ready to support toggling of filters just yet (I need to plumb in invalidation and redraw) but once that's there, for sure we want this in the UI.